gcn_classic_to_json/notices/SWIFT_BAT_GRB_POS_ACK/__init__.py

```diff
@@ -1,10 +1,79 @@
+import numpy as np
+
 from ... import utils
 
 
 def parse(bin):
+    bin[15]  # Unused. According to docs: '4 bytes for the future'
+    bin[19]  # Unused. Flags are either internal or equivalent to bin[18]
+    bin[26:36]  # Unused. According to docs: '40 bytes for the future'
+    bin[36]  # Unused. Flags Equivalent to bin[18]
+    bin[38]  # Unused. Sun/Moon parameters
+
+    integ_time = bin[14] * 16 / 1000
+
+    lat, lon = bin[16:17].view(">i2")
+
+    soln_status_bits = np.flip(np.unpackbits(bin[18:19].view(dtype="u1")))
+
+    flag_descriptions = {
```

Comment:
Lift this definition to module scope so that the dict is reused across invocations.

Reply:
This definition across all notice types, for example in HETE it's defined as `wxra2` (RA).

However, I think misc_flags_bits can be lifted

Reply:
Please lift it out of this function to the level of the same file.